### ISSUE TITLE
Add support .net standard 1.5 for CacheManager.SystemRuntimeCaching nuget

### ISSUE THREAD

**Issue by Ciget** [MichaCo/CacheManager#219]
Hi Micha,

Thanks a lot for your huge effort in order to make this package public.

Want to ask about your plans to add support of .net standard 1.5 or even newer for nuget CacheManager.SystemRuntimeCaching?

Thanks

**Comment by MichaCo** [comment]
Yes, System.Runtime.Caching was added back to netstandard2.0 I think. That target will be added soon
